pyecharts_snapshot/main.py

```diff
@@ -13,36 +12,50 @@
 else:
     from io import BytesIO
 
-PHANTOMJS_EXE = "phantomjs"
-NOT_SUPPORTED_FILE_TYPE = "Do not support file type %s"
+PHANTOMJS_EXEC = "phantomjs"
+NOT_SUPPORTED_FILE_TYPE = "Not supported file type '%s'"
 DEFAULT_DELAY = 1.5
 
 
 def main():
-    if len(sys.argv) < 2 or len(sys.argv) > 4:
-        raise sys.exit(-1)
-
-    try:
-        does_phantomjs_exist()
-    except OSError:
-        print("No phantomjs found in your path. Please install it!")
-        sys.exit(-1)
+    help = '''Usage:   snapshot {input file} {output type:[png|jpeg|gif|pdf]} {delay in seconds}
```

Comment:
help text is constant and never changes. hence it can be made as global constant.